src/sentry/rules/filters/age_comparison.py

```diff
@@ -55,11 +59,12 @@ class AgeComparisonFilter(EventFilter):
     label = "The issue is {comparison_type} than {value} {time}"
     prompt = "The issue is older or newer than..."
 
-    def passes(self, event: GroupEvent, state: EventState) -> bool:
+    def _passes(self, first_seen: datetime) -> bool:
         comparison_type = self.get_option("comparison_type")
         time = self.get_option("time")
+
         try:
-            value = int(self.get_option("value"))
+            validated_value = int(self.get_option("value"))
```

Comment:
you can change this back to `value`. sorry I wasn't clear in the original comment. best practice I wouldn't overwrite the previously named `value` variable with the validated version. but now that you don't pull it out separately you can just keep it as `value`